mantidproject/mantid
# GetIPTS algorithm

This is a cleanup of an algorithm written by an instrument scientist on SNAP. There are no unit tests and the usage tests are commented out, because it requires ORNL servers to be reachable. The algorithm takes a run number and (optional) instrument to return the IPTS directory containing the run.

**To test:**
1. Be inside the ORNL firewall
2. Try some runs and see that the run is in that proposal.

_There is no associated issue_

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
